pom.xml

```diff
@@ -121,7 +121,7 @@
         <yarn.version>v1.5.1</yarn.version>
 
         <!--Project properties-->
-        <ddf.version>2.14.0-SNAPSHOT</ddf.version>
+        <ddf.version>2.14.2-SNAPSHOT</ddf.version>
```

Comment:
❗️Becuase this PR is against Alliance master, shouldn't this be the version of DDF master: `2.15.0-SNAPSHOT`?

Reply:
that's a good question. I didn't want to go too far from where it was, but I'm happy to change it to 2.15.0

Reply:
updated

Reply:
Fixed by https://github.com/codice/alliance/pull/704